lib/broccoli/ember-app.js

```diff
@@ -335,7 +335,7 @@ EmberApp.prototype._initVendorFiles = function() {
   }
 
   // don't crash if ember-cli-shims is not included
-  if (!bowerDeps['ember-cli-shims']) {
+  if (!ember && !bowerDeps['ember-cli-shims']) {
```

Comment:
I'm actually rather opposed to this check existing as it makes the failure for not including shims move to runtime where we can't control it as easily and manifests as:

> cannot find module 'ember'

Regardless, knowing the story makes it possible for me to address. Thank you!